### PR TITLE
36 gui account

### LINKED ISSUE #36: GUI: Account

### PR DESCRIPTION
**Pull request by AndHager**
Closes #36

### CODE REVIEW

File: ctt-frontend/src/app/components/account/account.component.ts [hunk 1]
```diff
@@ -1,12 +1,47 @@
...
+      this.setUsername(username)
+    );
+    this.accountService.getEmail((email: string) => this.setEmail(email));
+    this.accountService.isEmailVerified((isVerified: string) =>
+      this.setIsVerified(isVerified)
+    );
+  }
+
+  private setUsername(username: string): void {
+    console.log('username: ' + username);
```
Comment (walli545) [inline ctt-frontend/src/app/components/account/account.component.ts]:
```suggestion
```

File: ctt-frontend/src/app/components/account/account.component.ts [hunk 2]
```diff
@@ -1,12 +1,47 @@
...
+    this.username = username;
+  }
+
+  private setEmail(email: string): void {
+    console.log('email: ' + email);
```
Comment (walli545) [inline ctt-frontend/src/app/components/account/account.component.ts]:
```suggestion
```

File: ctt-frontend/src/app/components/account/account.component.ts [hunk 3]
```diff
@@ -1,12 +1,47 @@
...
+    this.email = email;
+  }
 
-  ngOnInit(): void {}
+  private setIsVerified(isVerified: string): void {
+    console.log('isVerified: ' + isVerified);
```
Comment (walli545) [inline ctt-frontend/src/app/components/account/account.component.ts]:
```suggestion
```

**Review by walli545 — CHANGES_REQUESTED** [review]

**Review by walli545 — APPROVED** [review]

**Review by electron2302 — COMMENTED** [review]
What are the 192 changes in the schema for? are they on purpose?

**Review by sschoetz — APPROVED** [review]

**Pull request merged**